bulk_query_gen/influxdb/influx_metaquery_common.go

```diff
@@ -0,0 +1,58 @@
+package influxdb
+
+import (
+	"fmt"
+	"time"
+
+	bulkQuerygen "github.com/influxdata/influxdb-comparisons/bulk_query_gen"
+)
+
+type InfluxMetaquery struct {
+	InfluxCommon
+}
+
+// queryInterval is currently not used, but may be used to include as a parameter in the future.
+func NewInfluxMetaqueryCommon(lang Language, dbConfig bulkQuerygen.DatabaseConfig, queriesFullRange bulkQuerygen.TimeInterval, queryInterval time.Duration, scaleVar int) bulkQuerygen.QueryGenerator {
+	if _, ok := dbConfig[bulkQuerygen.DatabaseName]; !ok {
+		panic("need influx database name")
+	}
+
+	return &InfluxMetaquery{
+		InfluxCommon: *newInfluxCommon(lang, dbConfig[bulkQuerygen.DatabaseName], queriesFullRange, scaleVar),
+	}
+}
+
+func (d *InfluxMetaquery) Dispatch(i int) bulkQuerygen.Query {
+	q := bulkQuerygen.NewHTTPQuery() // from pool
+	bulkQuerygen.MetaqueryDispatchAll(d, i, q, d.ScaleVar)
+	return q
+}
+
+// StandardMetaquery generates a query that returns a list of tag values for a
+// specific tag key. The InfluxQL query is very simple and is constant. The Flux
+// query is slightly more complicated and is intended to replicate an equivalent
+// query that would be generated by the InfluxDB UI for getting "all" the tag
+// values for a specific tag key.
+func (d *InfluxMetaquery) StandardMetaquery(qi bulkQuerygen.Query) {
+	var query string
+	if d.language == InfluxQL {
+		query = `SHOW TAG VALUES FROM "example_measurement" WITH KEY = "X" LIMIT 200`
+	} else {
+		query = fmt.Sprintf(`from(bucket: "%s") `+
+			`|> range(start: %s, stop: %s) `+
```

Comment:
You might want someone with more expertise in flux than me take a look at this query.

Reply:
This LGTM, `keep(columns:[tag]) |> group() |> distinct(column: tag)` is the pattern that we'd expect to trigger the Flux pushdown

Reply:
It's based on the queries that the UI uses for getting metadata, so I think it should be representative.